### ISSUE TITLE
udocker has some Script error!

### ISSUE THREAD

**Issue by rayanfer32** [indigo-dc/udocker#286]
```sh
colab@337ec1ea0641:~$ ./udocker run hello-world
Warning: check container content: c3a16215-82bd-37a4-be6a-22d60d7b007c
Warning: non-existing user will be created
Traceback (most recent call last):
  File "./udocker", line 8547, in <module>
    sys.exit(Main().start())
  File "./udocker", line 8535, in start
    exit_status = self.execute()
  File "./udocker", line 8518, in execute
    status = cmds[command](self.cmdp)     # executes the command
  File "./udocker", line 7782, in do_run
    status = exec_engine.run(container_id)
  File "./udocker", line 3244, in run
    self.select_proot()
  File "./udocker", line 3174, in select_proot
    if HostInfo().oskernel_isgreater((4, 8, 0)):
  File "./udocker", line 506, in oskernel_isgreater
    os_version = [int(x) for x in os_release.split('.')[0:3]]
ValueError: invalid literal for int() with base 10: '112+'
```

**Comment by jorge-lip** [comment]
Hi,
Which Linux distro and version are you using ?

**Comment by Rayanfer32** [comment]
> Hi,
> Which Linux distro and version are you using ?

Google Colab

**Comment by jorge-lip** [comment]
Fixed on branches devel for Python 2 and devel3 for Python 3

**Comment by jorge-lip** [comment]
Included in v1.3.0. https://github.com/indigo-dc/udocker/releases/tag/v1.3.0
